wildfly-jakartaee-ear-archetype/src/main/resources/archetype-resources/README.txt

```diff
@@ -19,8 +19,8 @@ If you don't use entity beans, you can delete "persistence.xml".
 ==========================
 
 JSF:
-The web application is prepared for JSF by bundling an empty "faces-config.xml" in "src\main\webapp\WEB-INF".
-In case you don't want to use JSF, simply delete this file.
+The web application is prepared for JSF 2.3 by bundling an empty "faces-config.xml" in "src\main\webapp\WEB-INF".
+In case you don't want to use JSF, simply delete this file and "src\main\webapp\beans.xml" and "src\main\java\Jsf23Activator.java"
```

Comment:
it's "src\main\webapp\WEB-INF\beans.xml" 
I personally prefer the / for directories. Is there any convention here?

Reply:
Changed.